### PR TITLE
Records need to be associated with the manifest source table

### PR DESCRIPTION
**Pull request by aroltsch**
Update to the v2 data model. We need to connect `records` to `manifest_sources` so we know where the document came from.

Reference: https://github.com/department-of-veterans-affairs/caseflow-efolder/blob/master/docs/v2/database_structure.md

### CODE REVIEW

File: app/models/manifest.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 class Manifest < ActiveRecord::Base
-  has_many :manifest_statuses
+  has_many :sources, class_name: "ManifestSource"
   has_many :user_manifests
-  has_many :records
+  has_many :records, through: :sources
```
Comment (tejans24) [inline app/models/manifest.rb]: would this still work if ManifestSource doesn't have any association to Records?

File: app/models/record.rb [hunk 2]
```diff
@@ -1,6 +1,6 @@
 class Record < ActiveRecord::Base
-  belongs_to :manifest
+  belongs_to :manifest_source
```
Comment (tejans24) [inline app/models/record.rb]: ManifestSource does need a association to record then?

Comment (aroltsch) [reply app/models/record.rb]: Yes!

File: app/models/manifest.rb (hunk 1 above)
Comment (aroltsch) [reply app/models/manifest.rb]: It will not work!

**Pull request merged**